### PR TITLE
Section for feature/rework proposal design docs.

### PR DESCRIPTION
**Pull request by TemporalOroboros**
Adds a section between space-station-14 and the hosting guide for design proposals.

Also ports the three general design doc guide pages from the hackmd to fill it out for now.

- [x] TODO: Update the pages to reflect no longer being on the hackmd.

### CODE REVIEW

**Comment by mirrorcult** [comment]
linkcheck is failing 
image

**Comment by TemporalOroboros** [comment]
Currently installing mdbook so I can see the problem.

File: src/en/general-development/tips/proposal-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+It depends on the scale, and pervasiveness of whatever you're thinking of proposing. If it's something like adding a single gun or a couple simple plants it probably won't. On the other hand if you are adding an entire subdepartment ala anomalies/xenoarchaeology, or something on the order of reworking the entirity of botany or chemistry it certainly will.
+
+A good rule of thumb if the new feature or rework you have in mind would require adding or reworking a page of the guidebook or one of the feature docs on this site then it probably needs a design doc. Same if the proposal could be accurately described as 'reworking the entirity of X'.
+
+## Will my design doc get accepted?
+
+No idea! What design proposals do or do not get in is determined by maintainer approval like normal PRs. If you can get at least one maint to decide that it sounds like a good idea then there's a good chance that it will get accepted. Pretty much any idea is going to need at least some critiquing before it gets merged so don't get discouraged!
+
+``` admonish tip title="Design Principles"
```
Comment (mirrorcult) [inline src/en/general-development/tips/proposal-guidelines.md:42]:
```suggestion
``` admonish tip "Design Principles"
```

File: src/en/general-development/tips/proposal-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+2. A high level summary of what the feature actually is, how the feature engages with players, and how the feature interacts with other major parts of the game.
+3. A more detailed summary of the proposed mechanics for the feature, how players are intended to interact with those mechanics, and how those mechanics involve or are involved with other parts of the game. You don't need to go into every specific case, it's enough to say that there should be chemicals filling specific roles and not detail the exact chemicals you
+
+These don't have to be discrete sections and shouldn't be. When you're detailing the mechanics it's probably a good idea to detail how the players will interact with those mechanics and how the way the players interact with those mechanics benefits the game. If you want to see examples of successful design documents all of the accepted, but unimplemented, design docs can be found in the "Design Proposals" section below. Design documents that actually got implemented eventually transmute into the feature docs in the "Space Station 14" section.
+
+``` admonish warning title="Don't"
```
Comment (mirrorcult) [inline src/en/general-development/tips/proposal-guidelines.md:25]:
```suggestion
``` admonish warning "Don't"
```

File: src/en/general-development/tips/proposal-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+If you are considering adding or reworking some major component of the game it's recommended that you submit a proposal before you actually start coding whatever it is you want to do. Pinning down exactly how your feature should work to mesh well with the rest of the game will save you a lot of headaches rewriting your PR for the ~~1st2nd4th~~ 7th time before it gets merged.
+
+## How do I make a proposal?
+
+Just follow [our guide](../../meta/guide-to-editing-docs.md) on editing any part of this site and PR your proposal as a page under the "Design Proposals" section found in the sidebar between "Space Station 14" and "Server Hosting".
+
+Proposals that get approved by a maintainer will be merged to this site, which is basically the green light for you or someone else to go ahead and implement it.
+
+``` admonish tip title="Unfinished Proposals"
```
Comment (mirrorcult) [inline src/en/general-development/tips/proposal-guidelines.md:11]:
```suggestion
``` admonish tip "Unfinished Proposals"
```

**Review by mirrorcult — CHANGES_REQUESTED** [review]

**Review by mirrorcult — APPROVED** [review]

**Pull request merged**